config/routes.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  root 'users#index'
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index, :show]
```

Comment:
| @shm04 Index | Microverse Index |
|-|-|
| ![image](https://github.com/shm04/blogApp/assets/93039359/5c387ace-0cef-4129-8948-ed2c4ca75abf) | ![image](https://github.com/shm04/blogApp/assets/93039359/5f4a9263-8d2f-4c2e-9087-2db3bb88fbaf) |
| ![image](https://github.com/shm04/blogApp/assets/93039359/6a6718c1-bd92-4fa1-9e59-6704fbef53e8) | ![image](https://github.com/shm04/blogApp/assets/93039359/099b45f6-f610-4f2c-8511-1c5b64a74b51) |

| @shm04 Show | Microverse Show |
|-|-|
| ![image](https://github.com/shm04/blogApp/assets/93039359/1d913203-c011-40cc-89e0-dac073a1b267) | ![image](https://github.com/shm04/blogApp/assets/93039359/54266ae1-cb5e-40ad-a331-e7492ac1cb5d) |
| ![image](https://github.com/shm04/blogApp/assets/93039359/5557c501-ca23-498f-ac90-839e9aed37e5) | ![image](https://github.com/shm04/blogApp/assets/93039359/e157624c-46ad-4fc2-83de-e338e36263dd) |

- [ ] Great work so far but it would be nice if you make your views look similar to the given wireframe 👍 You don't need to put a lot of styling but at least your views should look similar to the given wireframe 😊 to follow the project requirements 📝